libs/pavex/Cargo.toml

```diff
@@ -47,6 +47,9 @@ serde_html_form = "0.2"
 serde_json = "1"
 serde_path_to_error = "0.1"
 
+# URL encdoed body extractor
+form_urlencoded = "1.2.1"
```

Comment:
We already have `serde_html_form` in our dependency tree, we should use it instead of adding a new dep.